### PR TITLE
Widget drag tests

### PR DESCRIPTION
**Pull request by ranbena**
## What type of PR is this?
- [x] Other - Tests

## Description
Widget dragging.

Screen Shot 2019-03-23 at 15 15 38

### CODE REVIEW

**Comment by arikfr** [comment]
Amazing. 

I'm really curious to see how these tests will hold when you do the refactor to React. 🤞

**Comment by gabrieldutra** [comment]
@ranbena is on fire :fire:

File: cypress/integration/dashboard/dashboard_spec.js [hunk 1]
```diff
@@ -179,6 +204,86 @@ describe('Dashboard', () => {
     });
   });
 
+  describe('Draggable widgets', () => {
+    const gutter = 15;
+    const colWidth = 200;
+
+    beforeEach(function () {
+      cy.viewport(gutter + 6 * colWidth, 800);
```
Comment (ranbena) [inline cypress/integration/dashboard/dashboard_spec.js]: Better suited for `before` but it gets reset at the end of each test.
@gabrieldutra is this a bug or a feature?

Comment (gabrieldutra) [reply cypress/integration/dashboard/dashboard_spec.js]: Not sure, but most of cypress state is reset on the end of each test, so my guess is that it's a feature. Also in the examples they never use `before` for this.

**Comment by ranbena** [comment]
Will wait with landing until #3622

**Pull request merged**